api/observability/tracing/ssh/session.go

```diff
@@ -342,3 +342,21 @@ func (s *Session) CombinedOutput(ctx context.Context, cmd string) ([]byte, error
 	output, err := s.Session.CombinedOutput(cmd)
 	return output, trace.Wrap(err)
 }
+
+// FileTransferRequestApprove sends a "file-transfer-request-response" ssh request
+// The response will contain an Approve bool which will approve or deny a requested file transfer
+func (s *Session) FileTransferRequestResponse(ctx context.Context, req FileTransferResponseReq) error {
```

Comment:
When I was reviewing the web PR, I realized that this might need a better name. The other method names are verb based while this one feels a bit odd compared to them.

Reply:
I tried thinking of something that would make sense for both like `ResponseToFileTransferRequest` but, idk I didn't like it. So I just split them into to, `ApproveFileTransferRequest` and `DenyFileTransferRequest`. They both take just the `requestID` now as a string, and the req is created inside the method and sets `Approved` appropriately. Maybe a bit verbose but much more readable and understandable